src/Authentication/Authenticators/AccessTokens.php

```diff
@@ -154,6 +154,19 @@ public function check(array $credentials): Result
 
         assert($token->last_used_at instanceof Time || $token->last_used_at === null);
 
+        // Is expired ?
+        if (
+            $token->expires !== null
+            && $token->expires->isBefore(
+                Time::now(),
```

Comment:
Did the tools add this, or is it extra?
```suggestion
                Time::now()
```

Reply:
Yes it's from the CS tool. Due to 'trailing_comma_in_multiline' rule, I guess.